myuw/dao/student.py

```diff
@@ -4,7 +4,7 @@
 determinations about user context in the case that a user is not
 enrolled for the current quarter
 """
-from uw_sws.enrollment import enrollment_search_by_regid
+from myuw.dao.enrollment import enrollment_search_by_regid
```

Comment:
please change enrollment_search_by_regid to "enrollment_search"

Reply:
Ah I see, we're caching withing the request. I'll update the method signatures to take a request instead and call enrollment_search.